### PR TITLE
#165 - Added list view option for the dashboard.

### PR DESCRIPTION
**Pull request by sangitamane**
[#165 ](https://github.com/pa11y/pa11y-dashboard/issues/165)  As discussed previously, added 2 simple list & grid view buttons to switch the view.

### CODE REVIEW

File: public/less/site.less [hunk 1]
```diff
@@ -235,6 +229,23 @@
   .dropdown-menu {
 	top: 25px;
   }
+	.gridview {
```
Comment (joeyciechanowicz) [inline public/less/site.less:232]: can you fix the indentation here?

File: public/less/variables-flatly.less [hunk 2]
```diff
@@ -233,7 +233,7 @@
 @navbar-default-bg: @brand-primary;
 @navbar-default-border: darken(@navbar-default-bg, 6.5%);
 @navbar-border-radius: @border-radius-base;
-@navbar-padding-horizontal: floor(@grid-gutter-width / 2); // ~15px
+@navbar-padding-horizontal: floor((@grid-gutter-width / 2)); // ~15px
```
Comment (joeyciechanowicz) [inline public/less/variables-flatly.less:236]: these changes that add an extra `(` seem redundant?

**Review by joeyciechanowicz — APPROVED** [review]
This all looks great, good addition! Just need to fix the indentation and extra brackets before merging

File: public/less/variables-flatly.less (hunk 2 above)
Comment (sangitamane) [reply public/less/variables-flatly.less:236]: @joeyciechanowicz, thanks for the review!  It was giving me an error while compiling less, therefore I added those extra brackets.
`ArgumentError: Error evaluating function `floor`: argument must be a number in`

File: public/less/site.less (hunk 1 above)
Comment (sangitamane) [reply public/less/site.less:232]: Done!

**Review by josebolos — APPROVED** [review]
Excellent work!

**Pull request merged**